typescript/ws-protocol/src/FoxgloveClient.ts

```diff
@@ -91,10 +91,10 @@ export default class FoxgloveClient {
     this.#ws.onopen = (_event) => {
       this.#emitter.emit("open");
     };
-    this.#ws.onmessage = (event: MessageEvent<ArrayBuffer | string>) => {
+    this.#ws.onmessage = (event: MessageEvent<ArrayBuffer | ArrayBufferView | string>) => {
       let message: ServerMessage;
       try {
-        if (event.data instanceof ArrayBuffer) {
+        if (event.data instanceof ArrayBuffer || ArrayBuffer.isView(event.data)) {
           message = parseServerMessage(event.data);
```

Comment:
I think this might be wrong, if it is an ArrayBufferView, then we need to respect the `byteOffset` and `byteLength`

Reply:
Do we need to make the corresponding change here?


https://github.com/foxglove/ws-protocol/blob/6382cea5da3f40ad98c898a651bbd3b28b39793b/typescript/ws-protocol/src/FoxgloveServer.ts#L391-L392

Reply:
yes, good catch